packages/docusaurus-theme-classic/src/theme/Navbar/styles.css

```diff
@@ -26,6 +26,24 @@
   -webkit-tap-highlight-color: transparent;
 }
 
+.toggle {
+  position: relative;
+  display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 10px;
```

Comment:
Use rem for size values. I think 0.625rem would be good.

Reply:
I'm a bit unsure about this since here it's actually matching the size of React Toggle. Then should I also change all the React Toggle styles to pixel values?